README.md

```diff
@@ -14,23 +14,62 @@
   </a>
 </p>
 
-[PipeRider](https://www.piperider.io/) ensures data reliability through automating constant data testing and monitoring.
 
-Key Features:
+# What's PipeRider?
+[PipeRider](https://www.piperider.io/) is a light-weight data quality tool so you can be confident of your data without writing tests for every single pipeline. 
 
-- Instant quality assessment in HTML
-  report: [sample report](https://piperider-github-readme.s3.ap-northeast-1.amazonaws.com/run-0.8.0/index.html)
-- Report
-  comparison: [sample comparsion report](https://piperider-github-readme.s3.ap-northeast-1.amazonaws.com/comparison-0.8.0/index.html)
-- [Extensible custom assertions](https://docs.piperider.io/data-quality-assertions/custom-assertions)
-- Works with existing [dbt](https://github.com/dbt-labs/dbt-core) projects
-- Automatic test recommendations
+**We're in an early stage, so [let us know](mailto:product@infuseai.io) if you have any questions, feedback, or need help installing PipeRider! :heart:**
+
+
+## Key Features
+* [Generate an HTML Report](how-to-guides/generate-report.md) featuring your data profile and data assertion test results ([interactive sample](https://piperider-github-readme.s3.ap-northeast-1.amazonaws.com/run-0.8.0/index.html))
+* [Compare two reports](how-to-guides/compare-reports.md) to understand how your data has changed over time ([interactive sample](https://piperider-github-readme.s3.ap-northeast-1.amazonaws.com/comparison-0.8.0/index.html))
+* Test your data with data assertions:
+  * Built-in [data assertions](data-quality-assertions/assertion-configuration.md)
+  * Extensible through [custom assertions](data-quality-assertions/custom-assertions.md)
+  * Auto-generated data assertions
+* Currently supports [Postgres](data-sources/postgres-connector.md), [Snowflake](data-sources/snowflake-connector.md), SQLite, [BigQuery](data-sources/bigquery-connector.md), [Redshift](data-sources/redshift-connector.md), [DuckDB](data-sources/duckdb-connector.md), [CSV](data-sources/csv-connector.md) and [Parquet](data-sources/parquet-connector.md).
+* Zero-config [support for dbt](dbt-integration/) projects
+* Automation through [GitHub Actions](how-to-guides/github-action.md), [save reports in S3](how-to-guides/aws-s3-+-github-ci.md)
+
+## Made for...
+**For data engineers**
+* :zap: 2 min install & set-up
+* :relieved: non-intrusive & open-source: install and use locally
+* :moneybag: fast & cheap: 10M rows, 8 columns takes only 16s to profile
+
+
+**For data analysts**
+* :bowtie: never waste time on analyzing wrong data: collects various metadata metrics such as freshness, uniqueness, distribution... [check all metrics](https://docs.piperider.io/data-profile-and-metrics/metrics)
+* :speech_balloon: communicate easily your data expectations by showing the report
+
+
+## Live Demo
+[![](https://i.imgur.com/WuFC4H6.png)](https://piperider-github-readme.s3.ap-northeast-1.amazonaws.com/run-0.7.0/index.html)
```

Comment:
Should it be `https://piperider-github-readme.s3.ap-northeast-1.amazonaws.com/run-0.7.0/index.html`?